server/segment_rpc.go

```diff
@@ -549,3 +555,11 @@ func genPayment(sess *BroadcastSession, numTickets int) (string, error) {
 
 	return base64.StdEncoding.EncodeToString(data), nil
 }
+
+func validatePrice(sess *BroadcastSession) bool {
```

Comment:
What do you think about return an error instead of a boolean here so all the details about price validation can be encapsulated in this function (rather than having `genPayment()` decide on an appropriate error message for an invalid price)?